cli/cmd/config.go

```diff
@@ -187,7 +183,7 @@ var configGetCmd = &cobra.Command{
 	Short:        "get config",
 	Long:         "Get the config for the given app, env, stack, and key",
 	SilenceUsage: true,
-	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+	PreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
This change is needed to have `happy config *` commands respect the `-v` flag from the root command.